logstash-core/lib/logstash/config/defaults.rb

```diff
@@ -6,6 +6,14 @@ module LogStash module Config module Defaults
 
   extend self
 
+  def input
+    "input { stdin { type => stdin } }"
+  end
+
+  def output
+    "output { stdout { codec => rubydebug } }"
```

Comment:
If not mistaken, the current default is without the codec. why do you add here the rubydebug codec? I know it provide more clear information but wondering if we should just output plain text without codec. what do you think?